Shared/mods/deathmatch/logic/lua/LuaBasic.cpp

```diff
@@ -15,17 +15,19 @@ namespace lua
     template <>
     std::string PopPrimitive<std::string>(lua_State* L, int& index)
     {
-        uint        uiLength = lua_strlen(L, index);
+        size_t      uiLength;
+        const char* str = lua_tolstring(L, index++, &uiLength);
         std::string outValue;
-        outValue.assign(lua_tostring(L, index++), uiLength);
+        outValue.assign(str, uiLength);
         return outValue;
```

Comment:
`return std::string(str, uiLength);` would be neater

Reply:
Thx